mp4parse/src/lib.rs

```diff
@@ -439,14 +439,188 @@ pub struct ProtectionSchemeInfoBox {
     pub tenc: Option<TrackEncryptionBox>,
 }
 
+/// Represents a userdata box 'udta'.
+/// Currently, only the metadata atom 'meta'
+/// is parsed.
+#[derive(Debug, Default, Clone)]
+pub struct UserdataBox {
+    pub meta: Option<MetadataBox>
```

Comment:
While calling the `udta` box `userdata` in the context is probably a reasonable decision, I'm not sure if the field for the nested `meta` box should be named `metadata`, since "metadata" can be ambiguous between track metadata, and metadata tags relating to the media the file or stream contains. Since the name of the box itself is `meta`, and it's a relatively self-explanatory FourCC, keeping it as `meta` makes sense, but naming it `metadata` would also be more consistent. 

If you have an opinion either way, would be happy to change it.